From e3c4df66e83f1540ec414b1cd5529a1aa2f04b8c Mon Sep 17 00:00:00 2001
From: MengLi <meng.li@windriver.com>
Date: Thu, 25 Jul 2019 09:51:11 +0800
Subject: [PATCH] arch: arm64: dts: enable Bluetooth sound card

TI BT-WF WL18xx module supports Bluetooth feature, it can roles as
a bridge between SoC audio interface and another BT audio device.
After enable the Bluetooth sound card, renesas-rcar platform
is able to  playback audio from another BT device.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/ulcb-kf.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/ulcb-kf.dtsi b/arch/arm64/boot/dts/renesas/ulcb-kf.dtsi
index 52a4db0636dd..24a927ff314f 100644
--- a/arch/arm64/boot/dts/renesas/ulcb-kf.dtsi
+++ b/arch/arm64/boot/dts/renesas/ulcb-kf.dtsi
@@ -165,7 +165,7 @@
 		label = "wl18xx";
 
 		dais = <&rsnd_port3>;
-		status = "disabled";
+		status = "okay";
 	};
 
 	lvds: lvds {
-- 
2.17.1

